handlers.py

```diff
@@ -554,20 +566,25 @@ async def handle_submission(request: Request) -> Response:
             stats.total_score += score.score
             stats.total_hits += score.n300 + score.n100 + score.n50
 
-            additive = score.score
-            if score.previous_score and score.status == 3:
-                additive -= score.previous_score.score
+            if score.mode.as_mode_int() in (1, 3):
```

Comment:
why only taiko & mania?

Reply:
that was my addition that i found in bancho.py basically geki and katu are added to total hits for mania and taiko (you had the comment explaining it there)

Reply:
ah right, didn't notice this was only adding geki & katu, nice